src/test/resources/templates/htmxRequest.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>htmx request</title>
+</head>
+<body>
+<!--/*@thymesVar id="#htmxRequest" type="HtmxRequest"*/-->
+<div id="htmxRequest" th:if="${#htmxRequest.isHtmxRequest()}">Is Htmx Request</div>
```

Comment:
I thought about shortening it to just `htmx` instead of `htmxRequest`, but decided the clarity that it is the same as the injected `HtmxRequest` was preferred.